content/version_control/git/git.tex

```diff
@@ -0,0 +1,129 @@
+
+\documentclass{beamer}
+\usepackage{changepage}
+\usetheme{default}
+
+\title{Git}
+
+\author{Kiran Vasudev\inst{1}}
+
+\institute[Universities of Somewhere and Elsewhere] 
+{
+  \inst{1}
+  Hochschule Bonn-Rhein-Sieg
+
+}
+
+\begin{document}
+
+\begin{frame}
+  \titlepage
+\end{frame}
+
+\begin{frame}{Outline}
+  \tableofcontents
+\end{frame}
+
+\begin{frame}{Introduction}
+\section{Introduction}
+  \begin{itemize}
+  \item {
+    Most widely used Version Control System(VCS)
+  }
+  \item {
+    Takes snapshots of the system.
+  }
+  \end{itemize}
+\begin{figure}
+	\includegraphics[scale=0.3]{images/git}
+	\caption{Different versions in the form of snapshots\cite{git-basics}}
+\end{figure}
+\end{frame}
+
+\begin{frame}{Main states of a Git repository}
+\section{Main states of a Git repository}
+  \begin{itemize}
+  \item {
+    Working directory
+  }
+  \item {   
+    Staging Area
+  }
+  \item {   
+    .git directory (repository)
+  }
+  \end{itemize}
+	\begin{figure}
+		\includegraphics[scale=0.3]{images/areas}
+		\caption{Working directory, staging area and the .git directory\cite{git-basics}}
+	\end{figure}
+\end{frame}
+
+\begin{frame}{States of files in your working directory}
+\section{States of files in your working directory}
+	\begin{itemize}
+		\item Files can either be tracked or untracked.
+	\end{itemize}
+	\begin{figure}
+		\includegraphics[scale=0.4]{images/lifecycle}
+		\caption{Lifecycle of the status of files\cite{recording-changes-git}}
+	\end{figure}
+\end{frame}	
+
+\begin{frame}[allowframebreaks]{Some important commands}
+\section{Some important commands}
```

Comment:
I think it may be more useful if we follow git pro's structure and at least group the commands by what their function is.